src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -220,38 +226,47 @@ public void clickAddSectionLevelPrivilegesLink(int instrNum) {
         this.addSectionLevelPrivilegesLink(instrNum).click();
     }
     
+    public WebElement sectionLevelPanel(int instrNum, int sectionLevelNum) {
```

Comment:
Suggestion: perhaps rename `sectionLevelNum` to `sectionLevelPanelIndex`?
Mainly it's the `Num` that is throwing me off because `Num` usually refers to a count of something